config/services.yaml

```diff
@@ -18,6 +18,9 @@ parameters:
         - { name: 'composerlock', label: 'composer.lock', route: "pugx_badge_composerlock" }
         - { name: 'gitattributes', label: '.gitattributes', route: "pugx_badge_gitattributes" }
 
+    redis.reader.class: 'Snc\RedisBundle\DependencyInjection\SncRedisExtension'
+    pugx.poser.class: 'PUGX\Poser\Poser'
```

Comment:
Sorry, why do you define these services? You can not use the autowire?

Reply:
`redis.reader.class` probably will go away with @daeronmalnwe review, while `pugx.poser.class` is for testing purpose, in `services_test.yml` I've specified a PoserStub in it